README.md

```diff
@@ -0,0 +1,76 @@
+![Project Banner](https://github.com/PikaDude/Animal-Crossing-Music-Extension/blob/master/docs/banner.png)
+
+# Animal Crossing Music (Revamp) 🍀
```

Comment:
I propose to call this a 'Revamp' for the time being but I'm not sold on the name. I still hope we can transfer all these changes into the original project!

Reply:
Maybe "Revived"?

Reply:
I will change it to "Revived" for now 👍